### PR TITLE
[ASM] EXPANDR-11761 - Update Remediation Guidance

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-11761](https://jira-dc.paloaltonetworks.com/browse/EXPANDR-11761)

## Description
Update playbook to set priority, rule description, and category fields even if remediation guidance is not provided from API endpoint.

### CODE REVIEW

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , looks good :shipit:

**Review by YairGlik — APPROVED** [review]

**Comment by YairGlik** [comment]
Hi @BigEasyJ  I see the PR is still a draft and has already been approved by @johnnywilkes , if you're done you can move it to open

**Comment by BigEasyJ** [comment]
@YairGlik ready to for merge

**Comment by YairGlik** [comment]
Hi @BigEasyJ, I noticed there are some conflicts with the master branch, could you resolve them? Once that's done, I'll be happy to merge your changes.

**Comment by BigEasyJ** [comment]
@YairGlik fixed conflicts

**Comment by YairGlik** [comment]
@BigEasyJ It looks like the build didn’t run correctly.
Please try pulling from the master branch again.

**Comment by ShirleyDenkberg** [comment]
@YairGlik @johnnywilkes Doc review completed.

**Comment by RotemAmit** [comment]
> @RotemAmit is there a way to bypass or remove the `license/cla` that popped up. I have already signed but it is not showing up. image

Hi @BigEasyJ,
Did @johnnywilkes also resign the CLA?

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance.yml
Comment (idovandijk) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance.yml:258]: It's safer to use the `SetIfEmpty` transformer for such cases, please use it instead

**Review by idovandijk — CHANGES_REQUESTED** [review]
Almost there, see my comment below

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance.yml
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Guidance.yml:258]: [96eac55](https://github.com/demisto/content/pull/37244/commits/96eac559ac4b5aa1268bd54ec319624595c7bfd8)

**Review by idovandijk — APPROVED** [review]
Nice

**Comment by Shellyber** [comment]
For the Reviewer: Trigger build request has been accepted for this contribution PR.

**Comment by Shellyber** [comment]
For the Reviewer: Successfully created a pipeline in GitLab with url: https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/pipelines/1778373

**Pull request merged**
